Tools/autotest/quadplane.py

```diff
@@ -78,6 +78,147 @@ def get_disarm_delay(self):
     def set_autodisarm_delay(self, delay):
         self.set_parameter("LAND_DISARMDELAY", delay)
 
+    def test_airmode(self):
+        """Check that plane.air_mode turns on and off as required"""
+        self.progress("########## Testing AirMode operation")
+        self.set_parameter("AHRS_EKF_TYPE", 10)
+        self.change_mode('QSTABILIZE')
+        self.wait_ready_to_arm()
+
+        """SPIN_ARM and SPIN_MIN default to 0.10 and 0.15"""
+        """when armed with zero throttle in AirMode, motor PWM should be at SPIN_MIN"""
+        """If AirMode is off, motor PWM will drop to SPIN_ARM"""
+
+        """Make sure that SERVO5 is Motor1 (default)"""
+        Motor1_value = 33
+        if (self.get_parameter('SERVO5_FUNCTION') != Motor1_value):
+            raise PreconditionFailedException("SERVO5_FUNCTION not %d" % Motor1_value)
+
+        """Make sure that flightmode channel is 5 (default)"""
+        default_fltmode_ch = 5
+        if (self.get_parameter("FLTMODE_CH") != default_fltmode_ch):
+            raise PreconditionFailedException("FLTMODE_CH not %d" % default_fltmode_ch)
+
+        """When disarmed, motor PWM will drop to min_pwm"""
+        #min_pwm = self.get_parameter('SERVO5_MIN')
+        """SERVO5_MIN is not respected; it defaults to 1100, but disarmed PWM is 1000"""
```

Comment:
Comment needs adjustment.

Actually fiddling with `Q_THR_MIN_PWM` a little to make sure it is honoured might be nice.

Reply:
done